test/integration/component/test_separate_backup_from_snapshot.py

```diff
@@ -0,0 +1,308 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+""" P1 tests for Snapshots
+"""
+# Import Local Modules
+from nose.plugins.attrib import attr
+from marvin.cloudstackTestCase import cloudstackTestCase
+
+from marvin.lib.base import (Snapshot,
+                             Template,
+                             VirtualMachine,
+                             Account,
+                             ServiceOffering,
+                             DiskOffering,
+                             Volume)
+
+from marvin.lib.common import (get_domain,
+                               get_zone,
+                               get_template,
+                               list_events,
+                               list_volumes,
+                               list_snapshots,
+                               list_templates,
+                               list_virtual_machines,
+                               )
+
+from marvin.lib.utils import (cleanup_resources,
+                              format_volume_to_ext3,
+                              random_gen,
+                              is_snapshot_on_nfs,
+                              get_hypervisor_type)
+
+from marvin.cloudstackAPI import detachVolume
+import time
+
+
+class Services:
+
+    """Test Snapshots Services
+    """
+
+    def __init__(self):
+        self.services = {
```

Comment:
All these config data is already defined in some existing test config file (I think testdata.cfg), try reusing that instead of maintaining a local copy with the test